From 6d6d2e1e5f56dbd553c1141313be687acc7241ea Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:58 -0400
Subject: [redhat] kabi: Adding symbol local_bh_disable (kernel/softirq.c)

Message-id: <34168a80cd64cf5a08348f7b9e2884d24d45c31d.1410510773.git.poros@redhat.com>
Patchwork-id: 95961
O-Subject: [PATCH RHEL7.1 118/233] kabi: Adding symbol local_bh_disable (kernel/softirq.c)
Bugzilla: 1077335
RH-Acked-by: Don Zickus <dzickus@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    To disable bottom-half when sending certain packets (mainly control) to our slow-path

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/local_bh_disable
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/local_bh_disable
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/local_bh_disable

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/local_bh_disable b/redhat/kabi/kabi-module/kabi_ppc64/local_bh_disable
new file mode 100644
index 0000000..6d238de
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/local_bh_disable
@@ -0,0 +1,2 @@
+#1-
+0x3ff62317 local_bh_disable vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/local_bh_disable b/redhat/kabi/kabi-module/kabi_s390x/local_bh_disable
new file mode 100644
index 0000000..6d238de
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/local_bh_disable
@@ -0,0 +1,2 @@
+#1-
+0x3ff62317 local_bh_disable vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/local_bh_disable b/redhat/kabi/kabi-module/kabi_x86_64/local_bh_disable
new file mode 100644
index 0000000..6d238de
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/local_bh_disable
@@ -0,0 +1,2 @@
+#1-
+0x3ff62317 local_bh_disable vmlinux EXPORT_SYMBOL
-- 
1.7.1